index.html

```diff
@@ -236,5 +237,52 @@ <h3>Skills</h3>
         </div>
       </section>
     </main>
+    <!--Contact form begins here-->
+    <section class="contact-me--form">
+      <div class="form-bg--image"></div>
+      <div>
+        <h2 class="contact-me">Contact me</h2>
+        <div class="form-intro">
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/moqrojrw" method="post">
+          <input
+            type="text"
+            name="user_name"
+            class="contact--field"
+            id="name"
+            maxlength="30"
+            required
+            placeholder="Alier"
+          /><br /><br />
+          <input
+            type="email"
+            pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$"
+            name="user_email"
+            class="contact--field"
+            id="email"
+            maxlength="30"
+            placeholder="alierpm2022@gmail.com"
+            required
+          /><br /><br />
+          <textarea
+            name="user_message"
+            class="text-msg"
+            id="message"
+            cols="42"
+            rows="8"
+            placeholder="Write your message here"
+            maxlength="500"
+            required
+          ></textarea
+          ><br /><br />
+          <input type="submit" class="form-button" value="Get In Touch" />
```

Comment:
- [ ] please make sure to add font-family, font-size and color to your button in order to match Figma design 
-Yours 
![image](https://user-images.githubusercontent.com/61790587/177565638-3fea0ce2-3a69-4ef9-885d-3a03d0f24255.png)
-Figma 
![image](https://user-images.githubusercontent.com/61790587/177565656-b777a83d-e913-476a-b3b9-115a71f5a17c.png)